ui-kit.code-workspace

```diff
@@ -0,0 +1,123 @@
+{
+  "folders": [
+    {
+      "name": "✨ ui-kit",
+      "path": "./"
+    },
+    {
+      "name": "📦 Auth",
+      "path": "./packages/auth"
+    },
+    {
+      "name": "📦 Bueno",
+      "path": "./packages/bueno"
+    },
+    {
+      "name": "📦 Headless",
+      "path": "./packages/headless"
+    },
+    {
+      "name": "📦 Quantic",
+      "path": "./packages/quantic"
+    },
+    {
+      "name": "📦 Atomic",
+      "path": "./packages/atomic"
+    },
+    {
+      "name": "📦 Atomic for React",
+      "path": "./packages/atomic-react"
+    },
+    {
+      "name": "📦 Atomic for Angular",
+      "path": "./packages/atomic-angular"
+    },
+    {
+      "name": "📦 Atomic Hosted Page",
+      "path": "./packages/atomic-hosted-page"
+    },
+    {
+      "name": "📄 Atomic Storybook documentation",
+      "path": "./utils/atomic-storybook"
+    },
+    {
+      "name": "🚀 IIFE samples",
+      "path": "./packages/samples/iife"
+    },
+    {
+      "name": "🚀 Stencil samples",
+      "path": "./packages/samples/stencil"
+    },
+    {
+      "name": "🚀 React samples (Headless)",
+      "path": "./packages/samples/headless-react"
+    },
+    {
+      "name": "🚀 React samples (Atomic)",
+      "path": "./packages/samples/atomic-react"
+    },
+    {
+      "name": "🚀 Next.js samples (Atomic)",
+      "path": "./packages/samples/atomic-next"
+    },
+    {
+      "name": "🚀 Angular samples",
+      "path": "./packages/samples/angular"
+    },
+    {
+      "name": "🚀 Vue.js samples",
+      "path": "./packages/samples/vuejs"
+    },
+    {
+      "name": "⚙️ Release utilities",
+      "path": "./utils/release"
+    },
+    {
+      "name": "⚙️ NX Adapter",
+      "path": "./utils/nx-plugin"
+    }
+  ],
+  "settings": {
+    "jest.disabledWorkspaceFolders": [
+      "✨ ui-kit",
+      "📦 Atomic",
+      "📦 Atomic for React",
+      "📦 Atomic for Angular",
+      "📄 Atomic Storybook documentation",
+      "📦 Atomic Hosted Page",
+      "🚀 IIFE samples",
+      "🚀 Stencil samples",
+      "🚀 React samples (Headless)",
+      "🚀 React samples (Atomic)",
+      "🚀 Next.js samples (Atomic)",
+      "🚀 Angular samples",
+      "🚀 Vue.js samples",
+      "⚙️ Release utilities",
+      "⚙️ NX Adapter",
+    ],
+    "jest.autoRevealOutput": "off",
+    "npm.packageManager": "npm",
+    "files.exclude": {
+      "**/node_modules/": true,
+      "packages/": true,
+      "utils/": true
+    },
+    "search.exclude": {
+      "dist/": true
+    },
+    "prettier.useTabs": false,
+    "editor.defaultFormatter": "esbenp.prettier-vscode",
+    "[typescript]": {
+      "editor.formatOnSave": true
+    },
+    "[javascript]": {
+      "editor.formatOnSave": true
+    },
+    "[typescriptreact]": {
+      "editor.formatOnSave": true
+    },
+    "[postcss]": {
+      "editor.formatOnSave": true
+    },
```

Comment:
IMO, format on save is a really nice time saver. Got used to it thanks to Flutter.